arrowhead/arrowhead.rb

```diff
@@ -15,20 +14,30 @@ class Arrowhead
     },
   }
 
+
+class Arrowhead
+
+  attr_reader :shape, :region_info
+
   # FIXME: I don't have time to deal with this.
```

Comment:
Deal with it!

(Probably was fixed... If the comment makes no sense, it's probably stale and should be deleted. Stale comments are worse than no comments.)